cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java

```diff
@@ -180,27 +180,41 @@ private void loadDbSchema(String schemaPath) {
     }
   }
 
-  private void initDbTable() {
-    if (dbSchema.getTables() != null) {
-      if (jsonField == null) {
-        logger.log(Level.SEVERE, "loadDbSchema(): No primary table name, can not load");
-        return;
-      }
-      // Remove the json blob field name, usually ".jsonb", but in tests also
-      // ".user_data" etc.
-      String tname = this.jsonField.replaceAll("\\.[^.]+$", "");
-      for (Table table : dbSchema.getTables()) {
-        if (tname.equalsIgnoreCase(table.getTableName())) {
-          dbTable = table;
-          break;
-        }
+  enum InitDbTableResult {
+    TABLE_FOUND,
+    AUDIT_TABLE_FOUND,
+    NO_TABLES_SECTION,
+    NO_PRIMARY_TABLE_NAME,
+    NOT_FOUND,
+  };
+
+  InitDbTableResult initDbTable() {
+    if (dbSchema.getTables() == null) {
+      logger.log(Level.SEVERE, "loadDbSchema loadDbSchema(): No 'tables' section found");
+      return InitDbTableResult.NO_TABLES_SECTION;
+    }
+    if (jsonField == null) {
+      logger.log(Level.SEVERE, "loadDbSchema(): No primary table name, can not load");
+      return InitDbTableResult.NO_PRIMARY_TABLE_NAME;
+    }
+    // Remove the json blob field name, usually ".jsonb", but in tests also
+    // ".user_data" etc.
+    String tname = this.jsonField.replaceAll("\\.[^.]+$", "");
+    boolean isAuditTable = false;
+    for (Table table : dbSchema.getTables()) {
+      if (tname.equalsIgnoreCase(table.getTableName())) {
+        dbTable = table;
+        return InitDbTableResult.TABLE_FOUND;
       }
-      if (dbTable == null) {
-        logger.log(Level.SEVERE, "loadDbSchema loadDbSchema(): Table {0} NOT FOUND", tname);
+      if (tname.equalsIgnoreCase(table.getAuditingTableName())) {
+        isAuditTable = true;
```

Comment:
Would it be simpler to use `return InitDbTableResult.AUDIT_TABLE_FOUND;` here?

Reply:
Yes, changed.